core/src/main/resources/application.yml

```diff
@@ -114,7 +114,7 @@ cb:
     url: http://localhost:8087
     contextPath: /redbeams
   runtimes:
-    latest: "7.1.1"
+    latest: "7.2.0"
```

Comment:
?

Reply:
Oops I left it there accidentally. (I only could select the 7.2.0 templates from the UI after this change)